### PR TITLE
Map generation

### PR DESCRIPTION
**Pull request by Redfirebird22**
Created oro_calibrate package to easily build map using gmapping and teleop nodes. Then use save.sh script to save map in oro_calibrate directory.

### CODE REVIEW

File: oro_calibrate/package.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>oro_calibrate</name>
+  <version>0.0.0</version>
+  <description>The oro_calibrate package</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag -->
+  <!-- Example:  -->
+  <!-- <maintainer email="jane.doe@example.com">Jane Doe</maintainer> -->
+  <maintainer email="elderado@todo.todo">elderado</maintainer>
```
Comment (ell0ry) [inline oro_calibrate/package.xml]: @Redfirebird22 Could you change this to have your email>

File: oro_calibrate/package.xml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <!-- Url tags are optional, but multiple are allowed, one per tag -->
+  <!-- Optional attribute type can be: website, bugtracker, or repository -->
+  <!-- Example: -->
+  <!-- <url type="website">http://wiki.ros.org/oro_calibrate</url> -->
+
+
+  <!-- Author tags are optional, multiple are allowed, one per tag -->
+  <!-- Authors do not have to be maintainers, but could be -->
+  <!-- Example: -->
+  <!-- <author email="jane.doe@example.com">Jane Doe</author> -->
```
Comment (ell0ry) [inline oro_calibrate/package.xml:28]: Same here, please add yourself as the author!

File: oro_gazebo/launch/oro_house.launch [hunk 3]
```diff
@@ -1,5 +1,5 @@
 <launch>
-  <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
+  <arg name="model" default="waffle_pi" doc="model type [burger, waffle, waffle_pi]"/>
```
Comment (ell0ry) [inline oro_gazebo/launch/oro_house.launch:3]: Could you add a todo comment here and at all <arg name="model"... lines so we have a papertrail of the intent to remove this?

File: oro_navigation/launch/oro_navigation.launch [hunk 4]
```diff
@@ -3,7 +3,7 @@
   <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
   <arg name="map_file" default="$(find oro_navigation)/maps/map.yaml"/>
   <arg name="open_rviz" default="true"/>
-  <arg name="move_forward_only" default="false"/>
+  <arg name="move_forward_only" default="true"/>
```
Comment (ell0ry) [inline oro_navigation/launch/oro_navigation.launch]: Could you revert this? We want this argument to be false

File: oro_calibrate/launch/oro_calibrate_build.launch [hunk 5]
```diff
@@ -0,0 +1,12 @@
+<launch>
+	<arg name="model" default="waffle_pi" doc="model type [burger, waffle, waffle_pi]"/>
```
Comment (ell0ry) [inline oro_calibrate/launch/oro_calibrate_build.launch:3]: Same with comment in later launch files, could you add a todo comment here?

File: oro_gazebo/launch/oro_world.launch [hunk 6]
```diff
@@ -1,5 +1,5 @@
 <launch>
-  <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
+  <arg name="model" default="waffle_pi" doc="model type [burger, waffle, waffle_pi]"/>
```
Comment (ell0ry) [inline oro_gazebo/launch/oro_world.launch:3]: Same as previous comment.

File: oro_slam/config/frontier_exploration.yaml [hunk 7]
```diff
@@ -20,9 +20,10 @@ static:
   subscribe_to_updates: true
 
 explore_boundary:
-  resize_to_boundary: false
-  frontier_travel_point: middle
-  explore_clear_space: false
+  resize_to_boundary: true
+  #frontier_travel_point: middle
+  frontier_travel_point: closest
+  #explore_clear_space: false
```
Comment (ell0ry) [inline oro_slam/config/frontier_exploration.yaml]: Could you revert this to the default if we're not using explore_boundary?

File: oro_slam/launch/oro_frontier_exploration.launch [hunk 8]
```diff
@@ -1,6 +1,6 @@
 <launch>
   <!-- Arguments -->
-  <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
+  <arg name="model" default="waffle_pi" doc="model type [burger, waffle, waffle_pi]"/>
```
Comment (ell0ry) [inline oro_slam/launch/oro_frontier_exploration.launch:5]: Same as previous comment, todo here please.

File: oro_slam/launch/oro_slam.launch [hunk 9]
```diff
@@ -1,6 +1,6 @@
 <launch>
   <!-- Arguments -->
-  <arg name="model" default="$(env TURTLEBOT3_MODEL)" doc="model type [burger, waffle, waffle_pi]"/>
+  <arg name="model" default="waffle_pi" doc="model type [burger, waffle, waffle_pi]"/>
```
Comment (ell0ry) [inline oro_slam/launch/oro_slam.launch:5]: Same as previous comment, todo here please.

**Review by ell0ry — CHANGES_REQUESTED** [review]
Awesome job Ant! Just a few things to clean things up before merging this. Mostly just nits and comments.

**Comment by Redfirebird22** [comment]
@th3rring Okay I have gone through and added TODO comments to launch files related to oro_calibrate. I also resolved the other comments. Let me know if there are any other specific places you want me to add TODO comments or anything else you would like me to take care of. Otherwise, I will go ahead and merge.

**Review by ell0ry — APPROVED** [review]

**Comment by ell0ry** [comment]
Awesome job Ant! We'll merge this at the meeting.

**Pull request merged**